auto_selfcal.py

```diff
@@ -656,6 +667,40 @@
                      gaintable=applycal_gaintable[vis],\
                      interp=applycal_interpolate[vis], calwt=True,spwmap=applycal_spwmap[vis],\
                      applymode=applycal_mode[band][iteration],field=target,spw=selfcal_library[target][band][vis]['spws'])
+
+         ## Create post self-cal image using the model as a startmodel to evaluate how much selfcal helped
+         ##
+         os.system('rm -rf '+sani_target+'_'+band+'_'+solint+'_'+str(iteration)+'_post*')
+         tclean_wrapper(vislist,sani_target+'_'+band+'_'+solint+'_'+str(iteration)+'_post',
+                  band_properties,band,telescope=telescope,nsigma=selfcal_library[target][band]['nsigma'][iteration], scales=[0],
```

Comment:
@jjtobin I start using this PR to review the changes happening after the point I forked.

First question: in these changes from @psheehan, we removed the start model keyword in the prototype. Was this result of the discussion with Luke back in the f2f meeting? The PL version lacks this currently, I plan to get it in soon. This will certainly make some difference in the outcome.

Reply:
Yes, this was the result of discussions with Luke and Ryan back in the f2f meeting. We decided that it would be best to redo tclean using the same threshold rather than using the startmodel because there were times that the start model no longer did a good job of representing the data. To get some speed back, we then copy the _post images to be the starting point for the next solint (and continue cleaning from there) so that we don't have to have a whole additional tclean from scratch.

Reply:
Perfect. Thanks for the clarification! I have sent John a note that the PL version now has all recent changes from the  [jjtobin:main](https://github.com/jjtobin/auto_selfcal). So this is in!

Reply:
Excellent!